auth_token/config.py

```diff
@@ -15,6 +15,12 @@
     'MAX_TOKEN_AGE': 60 * 60 * 24 * 7 * 2,  # Max token expiration time (default: 2 weeks)
     'COUNT_USER_PRESERVED_TOKENS': 20,  # Maximum tokens that will be preserved per user
     'TAKEOVER_REDIRECT_URL': '/',  # The path where will be used after takeover another user token
+    'TWO_FACTOR_ENABLED': False,  # Two factor authentication is disabled
+    'TWO_FACTOR_REDIRECT_URL': 'login-code-verification/',  # The path the user is redirected to after successful two
+                                                            # factor authentication
+    'TWO_FACTOR_USER_CONTACT_FIELD_NAME': 'email',  # User field name which should be used for second part of two
+                                                    # factor authorization
+    'AUTHENTICATION_METHOD': '',  # Method to send the key for second part of authorization process to the user
```

Comment:
Maybe I would explicitly state that this is path to class, at first I didn't understand what should be set in this settings.

Reply:
Maybe I would rename the setting to `TWO_FACTOR_SENDING_FUNCTION` or something like that.